### PR TITLE
app-laptop/lphdisk: Bump EAPI, update homepage

### PR DESCRIPTION
**Pull request by ephemer0l**
Package-Manager: Portage-2.3.13, Repoman-2.3.3

### CODE REVIEW

File: app-laptop/lphdisk/lphdisk-0.9.1-r3.ebuild [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+LICENSE="Artistic"
+SLOT="0"
+KEYWORDS="~x86"
+IUSE=""
+
+DEPEND="sys-libs/lrmi"
+
+src_prepare() {
+	epatch "${FILESDIR}"/${P}-gentoo.patch
```
Comment (monsieurp) [inline app-laptop/lphdisk/lphdisk-0.9.1-r3.ebuild]: Nop. Please create a `PATCH` array and add this patch to it (outside this function).

File: app-laptop/lphdisk/lphdisk-0.9.1-r3.ebuild [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+}
+
+src_compile() {
+	tc-export CC
+	emake || die
+}
+
+src_install() {
```
Comment (monsieurp) [inline app-laptop/lphdisk/lphdisk-0.9.1-r3.ebuild]: This is not needed. Please remove this phase.

File: app-laptop/lphdisk/lphdisk-0.9.1-r3.ebuild [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+DEPEND="sys-libs/lrmi"
+
+src_prepare() {
+	epatch "${FILESDIR}"/${P}-gentoo.patch
+}
+
+src_compile() {
+	tc-export CC
+	emake || die
```
Comment (monsieurp) [inline app-laptop/lphdisk/lphdisk-0.9.1-r3.ebuild]: Please remove the `|| die` here.

File: app-laptop/lphdisk/lphdisk-0.9.1-r3.ebuild [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+inherit eutils toolchain-funcs
+
+DESCRIPTION="utility for preparing a hibernation partition for APM Suspend-To-Disk"
+HOMEPAGE="https://web.archive.org/web/20160422052247/http://www.procyon.com/~pda/lphdisk/"
+SRC_URI="http://www.procyon.com/~pda/lphdisk/${P}.tar.bz2"
+
+LICENSE="Artistic"
+SLOT="0"
+KEYWORDS="~x86"
+IUSE=""
```
Comment (monsieurp) [inline app-laptop/lphdisk/lphdisk-0.9.1-r3.ebuild]: Please remove this variable if it is empty.

File: app-laptop/lphdisk/lphdisk-0.9.1-r3.ebuild [hunk 5]
```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit eutils toolchain-funcs
```
Comment (monsieurp) [inline app-laptop/lphdisk/lphdisk-0.9.1-r3.ebuild]: You can remove `eutils` here.

**Comment by monsieurp** [comment]
@ephemer0l ping?

**Comment by mgorny** [comment]
Ping².

**Comment by mgorny** [comment]
I'm going to close this for inactivity. Feel free to reopen if you are around and still interested in doing it.

**Pull request closed without merging**